packages/pluggableWidgets/rich-text-web/src/utils/ckeditorPresets.ts

```diff
@@ -0,0 +1,156 @@
+    {
+        name: "basicstyles",
+        items: [
+            "Bold",
+            "Italic",
+            "Underline",
+            "Strike",
+            "Subscript",
+            "Superscript",
+            "-",
+            "CopyFormatting",
+            "RemoveFormat"
+        ]
+    },
+    {
+        name: "paragraph",
+        items: [
+            "NumberedList",
+            "BulletedList",
+            "-",
+            "Outdent",
+            "Indent",
+            "-",
+            "Blockquote",
+            "CreateDiv",
+            "-",
+            "JustifyLeft",
+            "JustifyCenter",
+            "JustifyRight",
+            "JustifyBlock",
+            "-",
+            "BidiLtr",
+            "BidiRtl",
+            "Language"
+        ]
+    },
+    { name: "links", items: ["Link", "Unlink", "Anchor"] },
+    { name: "insert", items: ["Image", "Table", "HorizontalRule", "Smiley", "SpecialChar", "PageBreak", "Iframe"] },
+    { name: "styles", items: ["Styles", "Format", "Font", "FontSize"] },
+    { name: "colors", items: ["TextColor", "BGColor"] },
+    { name: "tools", items: ["Maximize", "ShowBlocks"] },
+    { name: "about", items: ["About"] }
+];
+
+function SET_PRESET(type: "basic" | "standard" | "full"): CKEditorConfig {
+    const config: CKEditorConfig = {};
+    let toolbarGroup: Array<ToolbarGroup | string> = [];
+    switch (type) {
+        case "basic":
+            config.toolbarGroups = [...TOOLBAR_GROUP];
+            config.removeButtons =
+                "Source,Save,Templates,NewPage,ExportPdf,Preview,Print,Cut,Redo,Copy,Undo,Paste,PasteText,PasteFromWord,Find,Replace,SelectAll,Scayt,Form,Checkbox,Radio,TextField,Textarea,Select,Button,ImageButton,HiddenField,Strike,Underline,Subscript,Superscript,CopyFormatting,RemoveFormat,CreateDiv,Blockquote,JustifyLeft,BidiLtr,Image,Table,BidiRtl,JustifyCenter,JustifyRight,Language,JustifyBlock,HorizontalRule,SpecialChar,Smiley,PageBreak,Iframe,Styles,TextColor,BGColor,ShowBlocks,Maximize,Format,Font,FontSize,Anchor";
+            break;
+        case "standard":
+            toolbarGroup = [...TOOLBAR_GROUP];
+            toolbarGroup.splice(4, 0, "/");
+            config.toolbarGroups = toolbarGroup;
+            config.removeButtons =
+                "Save,Templates,NewPage,ExportPdf,Preview,Print,Find,Replace,SelectAll,Form,Checkbox,Radio,TextField,Textarea,Select,Button,ImageButton,HiddenField,Underline,Subscript,Superscript,CopyFormatting,CreateDiv,JustifyLeft,BidiLtr,BidiRtl,JustifyCenter,JustifyRight,Language,JustifyBlock,Smiley,PageBreak,Iframe,TextColor,BGColor,Font,FontSize,ShowBlocks";
+            break;
+        case "full":
+            toolbarGroup = [...TOOLBAR_GROUP];
+            toolbarGroup.splice(3, 0, "/");
+            toolbarGroup.splice(5, 0, "/");
+            toolbarGroup.splice(6, 0, "/");
+            config.toolbarGroups = toolbarGroup;
+            config.removeButtons = "";
+            /* temporary removed exportpdf*/
```

Comment:
It's not really clear to me what this mean:
- Why do we need to do this?
- Temporary until when?

Reply:
`exportpdf` toolbar item was presented on PM's document, but due to some additional library connections I hide it from the list If we decide to include that plugin I will bring it back, if not, i'll just remove the comment.

`exportpdf` requires some authentication token, which should be defined by user itself, so it can be advanced configuration @cjfhodges

Reply:
so this is temporary that it still needs to be fixed from our side? If so, please prefix the comment with `TODO:`